compiler/modules/print.llvm/common.bal

```diff
@@ -18,7 +18,7 @@ public type IntegralType IntType|PointerType;
 
 // Corresponds to LLVMPointerType function
 public function pointerType(Type ty, int addressSpace = 0) returns PointerType {
-    return { pointsTo: ty, addressSpace };
+    return {pointsTo: ty, addressSpace};
```

Comment:
Please use consistent spacing style which has space after `{` and before `}` when inline.

Reply:
Fixed